### DISCUSSION TITLE
Options to configure my dashboard are gone

### DISCUSSION THREAD

**Discussion by scrutinizer11** [community/community#64731] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It used to be possible to add sections to your personal dashboard via a settings menu. Now, there are only 2 immovable sections: "For you" and "Following". The settings menu is no more. Could you bring it back?

**Comment by salah-elraies** [comment]
I think you're asking about [this](https://github.com/settings/profile), if not please elaborate more

**Comment by scrutinizer11** [comment]
I had stopped by every setting before starting this thread.
